corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -448,6 +471,11 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         },
     });
 
+    var updateContinueButtonText = function (prevValue, newValue) {
```

Comment:
This ought to be part of the `CaseListView`, since that's where the continue button is. Then to give the `CaseView` access to it, you can pass the `CaseListView` as part of the `CaseView`'s `childViewOptions`.

There are some docs [here](https://commcare-hq.readthedocs.io/web_apps.html). We do something similar in case search, to give the search field views (QueryView) access to the full case search view (QueryListView): https://github.com/dimagi/commcare-hq/blob/d8ebdc04a9d9ea08f358cd695f93c501585ced2c/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js#L276